api/src/com/cloud/agent/api/to/S3TO.java

```diff
@@ -155,6 +161,7 @@ public int hashCode() {
         result = 31 * result + (secretKey != null ? secretKey.hashCode() : 0);
```

Comment:
hasCode() and toString methods should be at the end of the file. The last methods.